packages/compiler/src/plugins.js

```diff
@@ -77,6 +95,49 @@ export const processMyst = () => {
   };
 };
 
+function parsePanels (contents) {
```

Comment:
👍 I would probably move this function into its own file-- maybe `myst.ts`? I imagine we'll need a bunch of custom logic to handle myst.